app/uk/gov/hmrc/nisp/views/account.scala.html

```diff
@@ -20,16 +20,17 @@
 @import uk.gov.hmrc.nisp.models.{SPSummaryModel, SPChartModel}
 @import uk.gov.hmrc.nisp.services.ABService
 @import uk.gov.hmrc.nisp.models.enums.ABTest._
+@import uk.gov.hmrc.play.frontend.auth.AuthContext
+@import uk.gov.hmrc.nisp.controllers.auth.NispUser
 
 @(
     nino: String,
     spSummary: SPSummaryModel,
-    user: uk.gov.hmrc.nisp.controllers.auth.NispUser,
     abTest: Option[ABTest],
     currentChart: SPChartModel,
     forecastChart: SPChartModel,
     authenticationProvider: String
-)(implicit request: Request[_])
+)(implicit request: Request[_], authContext: Option[AuthContext] = None, user: NispUser)
```

Comment:
@andrewgee Is NispUser needed anymore? The AuthContext now contains a place for the principal's name.